assets/overlays/openstack-cinder/generated/hypershift/controller.yaml

```diff
@@ -94,12 +93,18 @@ spec:
         - --endpoint=$(CSI_ENDPOINT)
         - --cloud-config=$(CLOUD_CONFIG)
         - --cluster=${CLUSTER_ID}
+        - --with-topology=$(ENABLE_TOPOLOGY)
```

Comment:
The effect of this is that when set to false `CreateVolume` will not pull topology information from the CreateVolumeRequest even if it is present:

https://github.com/kubernetes/cloud-provider-openstack/blob/53a4505c3caa96365b12cfc7230a84e429744b7c/pkg/csi/cinder/controllerserver.go#L88-L101

and subsequently `AvailabilityZone` will always be empty in the cinder create call:

https://github.com/kubernetes/cloud-provider-openstack/blob/53a4505c3caa96365b12cfc7230a84e429744b7c/pkg/csi/cinder/controllerserver.go#L194-L204